include/vapor/RenderParams.h

```diff
@@ -433,6 +436,13 @@ class PARAMS_API RenderParams : public ParamsBase {
     static const string _yFieldVariableNameTag;
     static const string _zFieldVariableNameTag;
     static const string _constantOpacityTag;
+    static const string XOriginTag;
```

Comment:
There is no documentation for these public tags. Prior to making many of the tags (e.g. _constantOpacityTag) public they were only accessible via public setters and getters that had doxygen documentation. The new flags introduced either need to have setters and getters that are documented, or the member variables themselves should be documented with Doxygen.

Reply:
Got it.  Fixed.  This info belongs in our [coding conventions](https://github.com/NCAR/VAPOR/wiki/Vapor-Coding-Convention) so I added a Doxygen section that covers this.